<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabOpcUaClient</class>
 <widget class="QWidget" name="TabOpcUaClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>620</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <item>
    <widget class="QTableWidget" name="table_nodes">
     <property name="columnCount">
      <number>3</number>
     </property>
     <column>
      <property name="text">
       <string>NodeId</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>BrowseName</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightPanelLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_config">
       <property name="title">
        <string>Configuration</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_config">
        <item row="0" column="0">
         <widget class="QLabel" name="label_endpoint">
          <property name="text">
           <string>Endpoint:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_endpoint">
          <property name="text">
           <string>opc.tcp://localhost:4840</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_timeout">
          <property name="text">
           <string>Timeout(ms):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spin_timeout">
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>60000</number>
          </property>
          <property name="value">
           <number>5000</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_control">
       <property name="title">
        <string>Actions</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_control">
        <item>
         <widget class="QPushButton" name="btn_connect">
          <property name="text">
           <string>Connect/Disconnect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_browse">
          <property name="text">
           <string>Browse Nodes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_readValue">
          <property name="text">
           <string>Read Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_writeValue">
          <property name="text">
           <string>Write Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_subscribe">
          <property name="text">
           <string>Subscribe/Unsubscribe</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="list_endpoints">
       <property name="minimumHeight">
        <number>80</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="txt_localLog">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="minimumHeight">
        <number>160</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>